core/lib/zksync_core/src/house_keeper/blocks_state_reporter.rs

```diff
@@ -36,13 +36,6 @@ impl L1BatchMetricsReporter {
                     .unwrap(),
                 BlockStage::MetadataCalculated,
             ),
-            (
-                conn.blocks_dal()
-                    .get_last_l1_batch_number_with_witness_inputs()
-                    .await
-                    .unwrap(),
-                BlockStage::MerkleProofCalculated,
-            ),
```

Comment:
I have checked, and I did not see its usage here(where its supposed to be) - https://grafana.matterlabs.dev/d/0Y8DUff4k/era-general?orgId=1&refresh=1m&from=now-30d&to=now&viewPanel=63
So I thought its safe to remove this.

Reply:
The value of this metric could be viewed by editing PromQL for the dashboard (rn, it explicitly lists stages that should be included, and `merkle_proof_calculated` is not one of them). The metric looks stagnant on testnet / mainnet (e.g., it's 61,729 on the mainnet while the current L1 batch is approx. 344,000), which I guess confirms that it's outdated. I must warn that using `git blame` in this case is not the best option; I've just refactored metrics in the codebase to use the `vise` façade 🙃